src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.scss';
+import List from "./components/List/List";
+
+function App() {
+  return (
+    <div className="container mt-4">
+      <List  />
```

Comment:
If you use a linter, you can automatically fix little oddities like that duplicate space
```suggestion
      <List />
```